src/main/java/com/powsybl/openloadflow/network/LfNetwork.java

```diff
@@ -546,6 +546,7 @@ public GraphConnectivity<LfBus, LfBranch> getConnectivity() {
             getBranches().stream()
                     .filter(b -> b.getBus1() != null && b.getBus2() != null)
                     .forEach(b -> connectivity.addEdge(b.getBus1(), b.getBus2(), b));
+            connectivity.setMainComponentVertex(getSlackBus());
```

Comment:
@flo-dup can you please just take a look at this change and tell us if it is okay for you. Thanks!